<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Payslips</class>
 <widget class="QMainWindow" name="Payslips">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Payslips</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Generate for all months</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomQueryView" name="lstMonths"/>
        </item>
        <item>
         <widget class="QCheckBox" name="chkGenerate">
          <property name="text">
           <string>Generate for all employees</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomQueryView" name="lstEmployees">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="trvMonths">
        <column>
         <property name="text">
          <string>Month</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="trvEmployees">
        <column>
         <property name="text">
          <string>Roll No</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="cmdGenerate">
        <property name="text">
         <string>Generate Payslips</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chkGenerate</sender>
   <signal>toggled(bool)</signal>
   <receiver>lstEmployees</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>lstMonths</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>trvMonths</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkGenerate</sender>
   <signal>toggled(bool)</signal>
   <receiver>trvEmployees</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
